tests/test_time.py

```diff
@@ -130,17 +131,13 @@ def test_sec2hms(self):
                     [0, 0, 1],
                     [0, 0, 30],
                     [0, 59, 59])
-        with warnings.catch_warnings(record=True) as w:
-            warnings.filterwarnings(
-                'always', 'Number of seconds > seconds in day.*',
-                UserWarning, '^spacepy\\.time')
+        with spacepy_testing.assertWarns(
+                self, 'always',
+                'Number of seconds > seconds in day\. Try days keyword\.$',
```

Comment:
I guess my earlier question about raw strings applies here.  It might be good to highlight (somewhere) when and why raw strings need to be used in calling this.

Reply:
Same answer as above. I _am_ being inconsistent with myself here, partly because I was copying it from an equality test not a regex test. I can make it a raw if it reads better.